extensions/amp-story/1.0/page-advancement.js

```diff
@@ -366,7 +379,18 @@ export class ManualAdvancement extends AdvancementConfig {
       event.preventDefault();
     }
 
-    this.storeService_.dispatch(Action.TOGGLE_PAUSED, false);
+    this.resolveTouchstart();
+  }
+
+  /**
+   * Reverts the touchstart events' state (eg: when resuming from swipe in player)
+   * @public
+   */
+  resolveTouchstart() {
```

Comment:
Can we name this method with something that better describes what it does? It doesn't just reset the `touchstartTimestamp` but executes all the touchend logic. Anything like processTouchend executeTouchend or anything you want works with me.